scout/server/blueprints/omics_variants/templates/omics_variants/outliers.html

```diff
@@ -70,27 +70,29 @@
           {% for variant in variants %}
             <tr>
               <td class="text-end">
-                <div class="d-flex flex-row justify-content-between align-items-baseline">
+                <div class="d-flex flex-row justify-content-between">
```

Comment:
Nice with the grouping! Not super happy about the alignment within the cell when aln is not on baseline, but I guess it follows the other cells. It is perhaps mostly about the gene symbol ending up a bit high, due to being linked?

Reply:
I think the gene symbol has a formatting of its own in the gene_cell macro imported from variants/components.html. I'll see if I can align it a little in the middle without messing up the other variantS pages

Reply:
After the fix: 
<img width="242" alt="image" src="https://github.com/user-attachments/assets/bbbc2181-33f1-4a92-b0bb-a507fdaeb7ef">

Other variants look good!